charts/stable/nextcloud/templates/_configmap.tpl

```diff
@@ -158,15 +158,17 @@ nextcloud-config:
       {{ . }}
       {{- end }}
 
+# TODO: Replace locations with ingress
+# like /push, /.well-known/carddav, /.well-known/caldav
```

Comment:
Can we add a reason the current /push wasn;t working?
Afaik it was because interal traffic also need ot be taken into account, correct?

Reply:
Because traffic going between pods directly, even with a domain name attached, isn';t going through the ingress filters.

Reply:
I think it was because nginx was manipulating the url and ingress didn't.
I'll lookup the chat